Source/Core/Core/HW/EXI/EXI_DeviceBaseboard.cpp

```diff
@@ -1,21 +1,115 @@
-// Copyright 2013 Dolphin Emulator Project
+// Copyright 2025 Dolphin Emulator Project
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 #include "Core/HW/EXI/EXI_DeviceBaseboard.h"
 
-#include "Common/Assert.h"
-#include "Common/ChunkFile.h"
+#include <numeric>
+#include <string>
+
+#include <fmt/format.h>
+
+#include "Common/Buffer.h"
 #include "Common/CommonTypes.h"
+#include "Common/FileUtil.h"
+#include "Common/IOFile.h"
 #include "Common/Logging/Log.h"
 
+#include "Core/Boot/Boot.h"
+#include "Core/BootManager.h"
+#include "Core/ConfigManager.h"
+#include "Core/Core.h"
+#include "Core/HW/DVD/AMMediaboard.h"
+#include "Core/HW/DVD/DVDInterface.h"
+#include "Core/HW/EXI/EXI.h"
+#include "Core/HW/EXI/EXI_Device.h"
+#include "Core/HW/MMIO.h"
+#include "Core/HW/Memmap.h"
+#include "Core/HW/SI/SI.h"
+#include "Core/HW/SI/SI_Device.h"
+#include "Core/HW/WiimoteReal/WiimoteReal.h"
```

Comment:
This include shouldn't be needed. (I didn't check all of them, but this one jumped out as out of place.)